src/modules/fancyzones/tests/UnitTests/ZoneWindow.Spec.cpp

```diff
@@ -15,10 +31,12 @@ namespace FancyZonesUnitTests
 TEST_METHOD(TestDeviceId)
 {
     // Window initialization requires a valid HMONITOR - just use the primary for now.
-    HMONITOR pimaryMonitor = MonitorFromWindow(nullptr, MONITOR_DEFAULTTOPRIMARY);
-    winrt::com_ptr<IZoneWindow> zoneWindow = MakeZoneWindow(nullptr, Mocks::Instance(), pimaryMonitor, L"SomeRandomValue", L"MyVirtualDesktopId", false);
-    // We have no way to test the correctness, just do our best and check its not an empty string.
-    Assert::IsTrue(zoneWindow->DeviceId().size() > 0);
+	HMONITOR pimaryMonitor = MonitorFromWindow(HWND(), MONITOR_DEFAULTTOPRIMARY);
```

Comment:
The tab @yuyoyuppe was referring to.

Reply:
Great point.   Thanks @yuyoyuppe  and @enricogior.  I've auto-formated the file in question, (and set tabs to spaces in my editor).